editor/src/messages/tool/tool_messages/select_tool.rs

```diff
@@ -168,10 +188,37 @@ impl LayoutHolder for SelectTool {
 		widgets.push(self.pivot_widget(self.tool_data.selected_layers_count == 0));
 
 		// Align
-		let disabled = self.tool_data.selected_layers_count < 2;
+		let align_to_artboard = self.tool_data.align_to_artboard;
+		let align_link_selected = self.tool_data.align_link_selected;
+		let disabled = self.tool_data.selected_layers_count == 0 || (!align_to_artboard && self.tool_data.selected_layers_count < 2);
 		widgets.push(Separator::new(SeparatorType::Unrelated).widget_holder());
-		widgets.extend(self.alignment_widgets(disabled));
-		widgets.push(PopoverButton::new("Align", "Coming soon").disabled(disabled).widget_holder());
+		widgets.extend(self.alignment_widgets(disabled, align_to_artboard, align_link_selected));
+		widgets.push(
+			// TODO: This button should be hidden when the canvas is infinite.
+			PopoverButton::new("Align", "Change alignment properties")
+				.options_widget(vec![
+					LayoutGroup::Row {
+						widgets: vec![
+							CheckboxInput::new(false)
```

Comment:
In order for the UI to display check mark correctly, you need to pass the current value of the input (here `tool_data.align_to_artboard`) into the CheckboxInput::new() function.

Same for the below input.

Reply:
Ah, I see. Where would be best to initialise the variables to true and false though?

Reply:
The default value is based on the default function for the select tool state (automatically derived at the moment).